tally_ho/libs/views/corrections.py

```diff
@@ -1,13 +1,64 @@
 from collections import OrderedDict
 
+from django.conf import settings
+from django.db.models import Q
 from django.forms import ValidationError
 from django.utils.translation import ugettext as _
 
 from tally_ho.apps.tally.models.candidate import Candidate
+from tally_ho.apps.tally.models.result_form_stats import ResultFormStats
 from tally_ho.apps.tally.models.result import Result
 from tally_ho.apps.tally.models.result_form import get_matched_results
 from tally_ho.libs.models.enums.entry_version import EntryVersion
 from tally_ho.libs.models.enums.race_type import RaceType
+from tally_ho.libs.permissions import groups
+
+
+def update_result_form_entries_with_de_errors(
+    de_1_suffix, de_2_suffix, post_data
+):
+    """Update result form stats entries that required DE corrections.
+
+    If correction value come from DE1, this means that the data entry error
+    was caused by DE2 clerk, and vise versa.
+
+    :param de_1_suffix: Suffix for identifying DE1 correction value
+    :param de_2_suffix: Suffix for identifying DE2 correction value
```

Comment:
Should we add post_data to the docstring?

Reply:
Nice catch, forgot to add it after doing a code refactor.